tests/src/Functional/SmsFrameworkGatewayAdminTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\sms\Functional;
+
+use Drupal\Core\Url;
+use Drupal\sms\Entity\SmsGateway;
+
+/**
+ * Tests manage event access page.
```

Comment:
The comment says "Tests manage event access page". We don't have a "Manage event" page. Maybe I misinterpreted the comment.